### PR TITLE
[New Rule] Suspicious PrintSpooler Point and Print DLL

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/640

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/privilege_escalation_printspooler_registry_copyfiles.toml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan = 30s
+[registry where
+ wildcard(registry.path, "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Print\\Printers\\*\\SpoolDirectory") and
+ wildcard(registry.data.strings, "C:\\Windows\\System32\\spool\\drivers\\x64\\4")]
+[registry where
+ wildcard(registry.path, "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Print\\Printers\\*\\CopyFiles\\Payload\\Module") and
+ wildcard(registry.data.strings, "C:\\Windows\\System32\\spool\\drivers\\x64\\4\\*")]
```
Comment (bm11100) [inline rules/windows/privilege_escalation_printspooler_registry_copyfiles.toml:36]: Are we still using the `wildcard` function or do we want to write it like below (haven't tested against real data, but validated the query syntax works)
```
sequence by host.id with maxspan = 30s
[registry where registry.path : "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Print\\Printers\\*\\SpoolDirectory") and
 registry.data.strings : "C:\\Windows\\System32\\spool\\drivers\\x64\\4"]
[registry where registry.path : "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Print\\Printers\\*\\CopyFiles\\Payload\\Module" and
 registry.data.strings : "C:\\Windows\\System32\\spool\\drivers\\x64\\4\\*")]
```

Comment (Samirbous) [reply rules/windows/privilege_escalation_printspooler_registry_copyfiles.toml:34]: tested only on 7.10 with wildcard(), not sure if this new syntax will work or not. 

image

**Review by bm11100 — APPROVED** [review]
LGTM 🌔

File: rules/windows/privilege_escalation_printspooler_registry_copyfiles.toml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "https://msrc.microsoft.com/update-guide/en-US/vulnerability/CVE-2020-1030",
+]
+risk_score = 74
+rule_id = "bd7eefee-f671-494e-98df-f01daf9e5f17"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan = 30s
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_printspooler_registry_copyfiles.toml:30]:
```suggestion
sequence by host.id with maxspan=30s
```

**Review by brokensound77 — APPROVED** [review]

**Pull request merged**